pkg/cmd/convert.go

```diff
@@ -62,12 +63,22 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
+	var err error
+
 	for _, file := range files {
-		data, err := ioutil.ReadFile(file)
-		if err != nil {
-			return nil, fmt.Errorf("unable to read file '%s': %s", file, err)
+		// Check if the passed resource points to STDIN or not
+		if file == "-" {
+			data, err = ioutil.ReadAll(os.Stdin)
```

Comment:
@surajssd I didn't find any method in cobra regarding this, do you know something that can help?